lib/paper_trail/queries/versions/where_object_changes.rb

```diff
@@ -28,13 +28,18 @@ def execute
               @version_model_class, @attributes
             )
           end
-          case @version_model_class.columns_hash["object_changes"].type
+          column_type = @version_model_class.columns_hash["object_changes"].type
+          case column_type
           when :jsonb
             jsonb
           when :json
             json
           else
-            text
+            raise UnsupportedColumnType.new(
+              method: "where_object_changes",
+              expected: "json/b",
```

Comment:
Some suggestion as above.

```suggestion
              expected: "json or jsonb",
```